.eleventy.js

```diff
@@ -49,48 +50,130 @@ const replaceContent = (item, searchValue, replaceValue) => {
  */
 module.exports = function (eleventyConfig) {
   addPreviewModeToEleventy(eleventyConfig, itemSetterCallback);
-
-  const wordpressImagePath = 'img/wordpress';
+  eleventyConfig.htmlTemplateEngine = "njk";
+  const wordpressImagePath = "img/wordpress";
+
+  eleventyConfig.addPlugin(cagovBuildSystem, {
+    processors: {
+      sass: {
+        watch: ["src/css/**/*"],
+        output: "_site_dist/index.css",
+        minify: true,
+        options: {
+          file: "src/css/index.scss",
+          includePaths: ["./src/css/sass"],
+        },
+      },
+      esbuild: [
+        {
+          watch: ["src/js/**/*"],
+          options: {
+            entryPoints: ["src/js/index.js"],
+            bundle: true,
+            minify: true,
+            outfile: "_site_dist/built.js",
+          },
+        },
+      ],
+    },
+  });
 
   eleventyConfig.addPassthroughCopy({ "wordpress/media": wordpressImagePath });
-
+  eleventyConfig.addPassthroughCopy({ "src/img": "img" });
+  eleventyConfig.addPassthroughCopy({ "src/pdf": "pdf" });
+  eleventyConfig.addPassthroughCopy({ "src/css/fonts": "fonts" });
+
+  // eleventyConfig.addFilter("dateformat", function (dateString) {
```

Comment:
I feel like if we include commented out code we should explain why it's there.

Reply:
Yep, good point